core/common/src/main/java/alluxio/underfs/UnderFileSystemRegistry.java

```diff
@@ -114,6 +114,7 @@ public static UnderFileSystem create(String path, Object ufsConf) {
         return factory.create(path, ufsConf);
       } catch (Exception e) {
         errors.add(e);
+        LOG.warn("Failed to create ufs", e);
```

Comment:
Currently the factories should not overlap, so I don't think we should expect failures. It just seems strange that we aggregate the errors and log at the end and at the same time log them as they come in.